### PR TITLE
fix: skip basic auth for rhc+satellite

### PR DESCRIPTION
**Pull request by zhangqianqian**

### CODE REVIEW

**Comment by Archana-PandeyM** [comment]
please fix lint issue.

**Review by Archana-PandeyM — CHANGES_REQUESTED** [review]
please fix lint issue.

**Review by jirihnidek — COMMENTED** [review]
Thanks for the PR, Unfortunately, this PR does not satisfy general requirements on PRs and commits. Each commit has to include description of a change. The PR itself has to include description of provided changes. Lint errors are also not acceptable.

Please, don't do any changes in Go code.

I also don't understand why should we skip basic auth for satellite?

I'm considering closing this PR.

**Comment by jirihnidek** [comment]
@zhangqianqian do you aware of this PR: https://github.com/RedHatInsights/rhc/pull/121 ?

File: integration-tests/test_connect.py [hunk 1]
```diff
@@ -25,6 +25,8 @@ def test_connect(external_candlepin, rhc, test_config, auth):
     """Test if RHC can connect to CRC using basic auth and activation key,
     Also verify that yggdrasil service is in active state afterward.
     """
+    if "satellite" in test_config.environment and auth == "basic":
```
[the remark this answers is not in the document]
Comment (Archana-PandeyM) [reply integration-tests/test_connect.py:30]: please add some description of change and  reference bug in commit message as well

**Comment by Archana-PandeyM** [comment]
@jirihnidek just FYI-  https://issues.redhat.com/browse/RHEL-53436
@zhangqianqian will be adding the description to commit message against my review request above, no need to close the PR

**Comment by zhangqianqian** [comment]
Thanks for your review and suggestion, @Archana-PandeyM .

Hi, @jirihnidek , I did not change any Go code, and I don’t know why the lint errors happened in my PR. I will not do any changes in Go code.

Of course I know https://github.com/RedHatInsights/rhc/pull/121 which is listed in bug https://issues.redhat.com/browse/RHEL-53436.

As per Christian’s statement in bug https://issues.redhat.com/browse/RHEL-53436, supporting basic auth for satellite is not the goal of the first milestone on supporting Satellite. As the QE who executes the rhc testing with satellite, I would like to keep the test cases updated with the current testing goal(only activation key is supported for satellite for now).  When we start the next milestone on supporting satellite when basic auth is supported, I will modify the test cases again. All my actions on the test code are just make sure the test cases reflects the testing goals. If https://github.com/RedHatInsights/rhc/pull/121 is merged now, we also have the reason to close this PR.  If we can not make sure ttps://github.com/RedHatInsights/rhc/pull/121 can be merged before next test cycle, I prefer merge this PR first.

**Review by jirihnidek — APPROVED** [review]
ACK

**Pull request merged**